<?xml version="1.0"?>
<launch>
    <arg name="camera_topic" default="/image_topic" />
    <arg name="action_detected_topic" default="action_detected" />
    <arg name="action_validation_topic" default="action_validation" />
    <arg name="learning_topic" default="True"/>
    <node pkg="action_recognition" type="action_recognition.py" name="action_recognition_node" output="screen">
        <param name="camera_topic" type="string" value="$(arg camera_topic)" />
        <param name="action_detected_topic" type="string" value="$(arg action_detected_topic)" />
        <param name="action_validation_topic" type="string" value="$(arg action_validation_topic)" />
        <param name="learning_topic" type="bool" value="$(arg learning_topic)" />
    </node>
</launch>
